src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -397,9 +409,9 @@ You can also query for all existing streams. Here's an example:
 }
 ```
 
-## Export compression [#compression]
+## Understand export compression [#compression]
 
-Optionally, we can compress payloads before they are exported, though this is disabled by default. This can help avoid hitting your ingested data limit and save money on ingress costs.
+Optionally, we can compress payloads before they are exported, though this is disabled by default. This can help avoid hitting your ingested data limit and save money on egress costs.
```

Comment:
What's en egress cost?

Reply:
Egress costs are outlays related to sending data to other cloud providers. I think our readers will know about this.